GOVERNANCE.md

```diff
@@ -169,9 +181,18 @@ Provide a summary of the nominee's contributions. For example:
 Mention @nodejs/collaborators in the issue to notify other collaborators about
 the nomination.
 
-The nomination passes if no collaborators oppose it after one week. In the case
-of an objection, the TSC is responsible for working with the individuals
-involved and finding a resolution.
+The nomination passes if no collaborators explicitly oppose it after one week.
+In the case of an objection, the TSC is responsible for working with the
+individuals involved and finding a resolution. The TSC may, following
+typical TSC consensus seeking processes, choose to advance or block a
+nomination that has otherwise failed to reach a natural consensus or clear
+path forward even if there are outstanding objections.
+
+Explicit opposition would typically be signaled as some form of clear
+and unambiguous comment like, "I don't believe this nomination should pass"
+or use of the thumbs-down emoji reaction. _Ideally_ these would be paired
```

Comment:
Here I would say "these must be paired".

Reply:
If it is "must" instead of "would" or "should", then "Ideally" also needs to be removed.

Reply:
I don't think "must" is necessary here so long as it is clear that the objection can be overridden if the strong recommendation is not followed.